x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/CrossClusterEsqlRCS1UnavailableRemotesIT.java

```diff
@@ -0,0 +1,283 @@
+                }
+              ],
+              "remote_indices": [
+                {
+                  "names": ["points", "squares"],
+                  "privileges": ["read", "read_cross_cluster", "create_index", "monitor"],
+                  "clusters": ["my_remote_cluster"]
+                }
+              ]
+            }""");
+        assertOK(adminClient().performRequest(putRoleOnRemoteClusterRequest));
+    }
+
+    private void loadData() throws IOException {
+        Request createIndex = new Request("PUT", "points");
+        createIndex.setJsonEntity("""
+            {
+                "mappings": {
+                    "properties": {
+                      "id": { "type": "integer" },
+                      "score": { "type": "integer" }
+                    }
+                }
+            }
+            """);
+        assertOK(client().performRequest(createIndex));
+
+        Request bulkRequest = new Request("POST", "/_bulk?refresh=true");
+        bulkRequest.setJsonEntity("""
+            { "index": { "_index": "points" } }
+            { "id": 1, "score": 75}
+            { "index": { "_index": "points" } }
+            { "id": 2, "score": 125}
+            { "index": { "_index": "points" } }
+            { "id": 3, "score": 100}
+            { "index": { "_index": "points" } }
+            { "id": 4, "score": 50}
+            { "index": { "_index": "points" } }
+            { "id": 5, "score": 150}
+            """);
+        assertOK(client().performRequest(bulkRequest));
+
+        createIndex = new Request("PUT", "squares");
+        createIndex.setJsonEntity("""
+            {
+                "mappings": {
+                    "properties": {
+                      "num": { "type": "integer" },
+                      "square": { "type": "integer" }
+                    }
+                }
+            }
+            """);
+        assertOK(performRequestAgainstFulfillingCluster(createIndex));
+
+        bulkRequest = new Request("POST", "/_bulk?refresh=true");
+        bulkRequest.setJsonEntity("""
+            { "index": {"_index": "squares"}}
+            { "num": 1, "square": 1 }
+            { "index": {"_index": "squares"}}
+            { "num": 2, "square": 4 }
+            { "index": {"_index": "squares"}}
+            { "num": 3, "square": 9 }
+            { "index": {"_index": "squares"}}
+            { "num": 4, "square": 16 }
+            """);
+        assertOK(performRequestAgainstFulfillingCluster(bulkRequest));
+    }
+
+    private Request esqlRequest(String query) throws IOException {
+        XContentBuilder body = JsonXContent.contentBuilder();
+
+        body.startObject();
+        body.field("query", query);
+        body.field("include_ccs_metadata", true);
+        body.endObject();
+
+        Request request = new Request("POST", "_query");
+        request.setJsonEntity(org.elasticsearch.common.Strings.toString(body));
```

Comment:
nit: you can move the pull path to the imports, since you are only using one version of `Strings` in this test.